environment/learnrails/spec/controllers/contacts_controller_controller_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe ContactsControllerController, type: :controller do
+  describe 'create' do
+    it 'successfully creates a new contact' do
+      expect{
+          post :create, params: { :contacts => { :contact => "testcontact"} }
+        }.to change(ContactsController,:count).by(1)
+    
```

Comment:
I think that you need to check if Contact is created not ContactsController
something like
change(Contact, :count).by(1)